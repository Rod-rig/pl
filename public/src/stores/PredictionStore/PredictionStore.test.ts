import {PredictionStore} from './';

describe('PredictionStore', () => {
  const mockStore = new PredictionStore();

  it('should be initialized correctly', () => {
    expect(mockStore).toBeInstanceOf(PredictionStore);
  });

  it('should create correct object', () => {
    expect(mockStore.matches[0]).toMatchSnapshot();
  });
});
