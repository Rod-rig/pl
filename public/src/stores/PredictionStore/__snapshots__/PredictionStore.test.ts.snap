// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PredictionStore should create correct object 1`] = `
Object {
  "competitors": Array [
    Object {
      "abbreviation": "NEW",
      "country": "England",
      "country_code": "ENG",
      "id": "sr:competitor:39",
      "name": "Newcastle United",
      "qualifier": "home",
    },
    Object {
      "abbreviation": "TOT",
      "country": "England",
      "country_code": "ENG",
      "id": "sr:competitor:33",
      "name": "Tottenham Hotspur",
      "qualifier": "away",
    },
  ],
  "id": "sr:match:14735959",
  "scheduled": "2018-08-11T11:30:00+00:00",
  "season": Object {
    "end_date": "2019-05-13",
    "id": "sr:season:54571",
    "name": "Premier League 18/19",
    "start_date": "2018-08-10",
    "tournament_id": "sr:tournament:17",
    "year": "18/19",
  },
  "start_time_tbd": false,
  "status": "not_started",
  "tournament": Object {
    "category": Object {
      "country_code": "ENG",
      "id": "sr:category:1",
      "name": "England",
    },
    "id": "sr:tournament:17",
    "name": "Premier League",
    "sport": Object {
      "id": "sr:sport:1",
      "name": "Soccer",
    },
  },
  "tournament_round": Object {
    "number": 1,
    "type": "group",
  },
  "venue": Object {
    "capacity": 52338,
    "city_name": "Newcastle upon Tyne",
    "country_code": "ENG",
    "country_name": "England",
    "id": "sr:venue:765",
    "map_coordinates": "54.975033,-1.615061",
    "name": "St James' Park",
  },
}
`;
